kubernetes/README.md

```diff
@@ -1,4 +1,4 @@
-# kubernetes.client
```

Comment:
the drop of `kubernetes` weird. I couldn't reproduce it locally. Also I noticed that this commit includes a file `kubernetes/swagger.json` which we don't commit anymore. Could you double check and share the `kubernetes-client/gen` version, `openapi-generator` version and your OS version being used?

Reply:
The `kubernetes-client/gen` version and  `openapi-generator` version should be pinned since the update scripts takes it from upstream.

I am suspecting something which may cause that import issue, but not sure why the issue is occurring though. If you see https://github.com/kubernetes-client/gen/blob/master/openapi/python.sh#L63-L65, those lines find and replace all instances of `import client` and `from client` to `import kubernetes.client` and `from kubernetes.client` respectively. That line should run always when the generator script is executed.

Reply:
> If you see https://github.com/kubernetes-client/gen/blob/master/openapi/python.sh#L63-L65, those lines find and replace all instances of import client and from client to import kubernetes.client and from kubernetes.client respectively

Nice catch!